custodial-copy-backend/src/main/scala/uk/gov/nationalarchives/custodialcopy/OcflService.scala

```diff
@@ -19,11 +21,15 @@ import scala.jdk.FunctionConverters.*
 
 class OcflService(ocflRepository: MutableOcflRepository, semaphore: Semaphore[IO]) {
 
+  given Logger[IO] = Slf4jLogger.getLogger[IO]
+
+  private def onError(err: Throwable): IO[Unit] = Logger[IO].error(err)(err.getMessage) >> semaphore.release
```

Comment:
Maybe `logError` or `logErrorAndRelease` rather than `onError`?

Reply:
`logErrorAndRelease` is ok, I've changed it.